src/pages/project/remix.js

```diff
@@ -25,9 +26,12 @@ module.exports = {
           uri: `/users/${this.state.params.user}/projects/${this.state.params.project}`
         }, (err, moreData) => {
           if (err) {
+            window.Platform.trackEvent('Remix', 'Remix Error', err);
             return console.error('Error remixing project', err);
           }
 
+          window.Platform.trackEvent('Remix', 'Remix', projectID);
```

Comment:
Could we call this `Remix Success` to make it easier to understand in the reporting. Or `Remix Started` if that better describes what has happened?

```
window.Platform.trackEvent('Remix', 'Remix Success', projectID);
```